src/org/zaproxy/zap/extension/invoke/InvokeAppWorker.java

```diff
@@ -66,6 +69,7 @@ protected Void doInBackground() throws Exception {
 		String site = "";	// e.g. http://localhost:8080/
 		String postdata = "";	// only present in POST ops
 		String cookie = "";		// from the request header
+		int msgid = msg.getHistoryRef().getHistoryId();
```

Comment:
This will lead to a `NullPointerException` when the `HttpMessage` does not have a `HistoryReference` (e.g. Manual Request Editor).